cpplib/Tests/GeometryTest.cpp

```
@@ -783,7 +785,19 @@ TEST_F(PolygonTest, EdgeCases) {
+TEST(Voronoi, CellConstruction) {
+	// Insufficient number of points
+	voronoi::Cell cell(Point<FloatingPointType>(0.2, 0.3, 0.4), 1);
+	constexpr Plane<FloatingPointType> plane(Point<FloatingPointType>(0.5, 0.5, 0.5), Point<FloatingPointType>(-1, -1, -1));
+	cell.clipByPlaneAndAddNewFace(plane, 2, 13);
+
+
+}
+
 // ==================== TESTS FOR Voronoi ====================
+// TODO: Tests disabled until Voronoi rework
 class VoronoiTest : public ::testing::Test {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: commonPoints, singlePoint

(cppcoreguidelines-pro-type-member-init)